person.rb

```diff
@@ -0,0 +1,24 @@
+# implementation person class
+class Person
+  def initialize(age, _parent_permission: true, name: 'Unknown')
+    @id = id
+    @name = name
+    @age = age
```

Comment:
- [ ] The @parent_permission instance variable should be initialized in the initialized method. Something like so:
`  @parent_permission = parent_permission`